src/modules/mahjong/QuadValidator/QuadValidator.ts

```diff
@@ -0,0 +1,51 @@
+import { CreateQuadResults } from '../types/MahjongTypes';
+import HandValidator from '../HandValidator/HandValidator';
+import PlayerHand from '../Hand/PlayerHand';
+import Tile from '../Tile/Tile';
+import TileFactory from '../Tile/TileFactory';
+
+class QuadValidator {
```

Comment:
Do we need quadvalidator or can we add it as a method to hand validator?

Reply:
We don't really need it as its own thing. I thought it should be separate as it's not really related to hand validation. Could be a utils func